rcaracaus/pledge-vegan-node
# Removed config from load files service [3](5)

Steps to test:

1. Go to every page on localhost:3000
2. Make sure that every page loads without a 404 error
3. Make sure every image and link works
